evap/student/forms.py

```diff
@@ -12,8 +12,45 @@ def __init__(self, label):
         super().__init__(label=label, required=False)
 
 
+class TextAnswerField(forms.CharField):
+
+    def __init__(self, related_question_id="", *args, **kwargs):
+        # pylint: disable=keyword-arg-before-vararg
+        self.related_question_id = related_question_id
+        kwargs['required'] = False
+        kwargs['widget'] = forms.Textarea(attrs={'related_question_id': self.related_question_id})
+        super().__init__(*args, **kwargs)
+
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        del kwargs['required']
```

Comment:
I'm not sure if I understand correctly what's happening here, but from what I understand, this is supposed to return information on how to construct the current instance. We `del kwargs['required']` here, because in the constructor, we will manually add it. Why isn't the same true for `kwargs['widget']`?

Reply:
you are right, the widget is as well always set in the init method. so we can remove it as well. i changed that now.